mon-pix/app/services/session.js

```diff
@@ -88,13 +90,12 @@ export default class CurrentSessionService extends SessionService {
   async _handleLocale(localeFromQueryParam = null) {
     const isUserLoaded = !!this.currentUser.user;
     const domain = this.currentDomain.getExtension();
-    const defaultLocale = 'fr';
 
-    if (domain === 'fr') {
-      await this._setLocale(defaultLocale);
+    if (domain === FRANCE_TLD) {
```

Comment:
Remarque ; j'ai peur que ça modifie encore beaucoup de tests si on remplace 

par 
```suggestion
if (this.currentDomain.isFranceDomain) {